bundles/mapping/mapmodule/domain/MaplayerGroup.js

```diff
@@ -102,12 +104,24 @@ Oskari.clazz.define('Oskari.mapframework.domain.MaplayerGroup',
         setName: function (name) {
             this.name = name;
         },
+        getLayers: function () {
+            return this.layers;
+        },
+        setLayers: function (layers) {
+            this.layers = layers;
+        },
         getOrderNumber: function () {
             return this.orderNumber;
         },
         setOrderNumber: function (orderNumber) {
             this.orderNumber = orderNumber;
         },
+        getRootId: function () {
+            return this.rootId;
+        },
+        setRootId: function (rootId) {
+            this.rootId = rootId;
+        },
```

Comment:
Looks like there is no code that calls set or get for rootId. Is there a reason for these?